README.md

```diff
@@ -61,13 +61,12 @@ import com.stripe.net.RequestOptions;
 public class StripeExample {
 
     public static void main(String[] args) {
-        RequestOptions requestOptions = (new RequestOptionsBuilder()).setApiKey("YOUR-SECRET-KEY").build();
         Map<String, Object> chargeMap = new HashMap<String, Object>();
         chargeMap.put("amount", 100);
         chargeMap.put("currency", "usd");
         chargeMap.put("source", "tok_1234"); // obtained via Stripe.js
         try {
-            Charge charge = Charge.create(chargeMap, requestOptions);
+            Charge charge = Charge.create(chargeMap);
```

Comment:
I also simplified this first example in favor of the new section.

Reply:
sgtm, but should we update the example to show how to set the API key globally then?

Reply:
Oops, yep that would be quite helpful wouldn't it? Fixed and commit amended.